src/components/FreeResponseReview/FreeResponseReview.tsx

```diff
@@ -0,0 +1,56 @@
+import React from "react";
+import { 
+  FreeResponseDefaultContainer, 
+  FreeResponseExpandedContainer, 
+  FreeResponseText, 
+  ReadMoreLink, 
+  COLLAPSED_HEIGHT 
+} from "./FreeResponseReview.styles";
+
+export const FreeResponseReview = ({ free_response, previewMode }: {
+  free_response?: string;
+  previewMode?: boolean;
+}) => {
+
+  const [expanded, setExpanded] = React.useState(false);
+  const [isOverflowing, setIsOverflowing] = React.useState(false);
+  const textRef = React.useRef<HTMLDivElement>(null);
+
+  const FreeResponseContainer = ({ children }: { children: React.ReactNode }) => {
+    return (
+      previewMode 
+        ? <FreeResponseExpandedContainer>{children}</FreeResponseExpandedContainer>
+        : <FreeResponseDefaultContainer>{children}</FreeResponseDefaultContainer>
+    );
+  };
+
+  React.useEffect(() => {
```

Comment:
Maybe this should use `useLayoutEffect` since it'll cause a re-render but I'm not 100% sure. It's possible that the result will just be incorrect at first with `useLayoutEffect`.